client/app/scripts/liveblog-edit/timeline.js

```diff
@@ -72,6 +81,96 @@ define([
+        'notify', 'gettext', 'asset', 'postsService', 'modal', 'blogService',
+        function(notify, gettext, asset, postsService, modal, blogService) {
+            return {
+                scope: {
+                    post: '=',
+                    onEditClick: '='
+                },
+                replace: true,
+                restrict: 'E',
+                templateUrl: 'scripts/liveblog-edit/views/timeline-post.html',
+                link: function(scope, elem, attrs) {
+                    scope.toggleMultipleItems = function() {
+                        scope.post.show_all = !scope.post.show_all;
+                    };
+
+                    scope.removePost = function(post) {
+                        postsService.remove(post).then(function(message) {
+                            notify.pop();
+                            notify.info(gettext('Post removed'));
+                        }, function() {
+                            notify.pop();
+                            notify.error(gettext('Something went wrong'));
+                        });
+                    };
+
+                    scope.askRemovePost = function(post) {
+                        modal.confirm(gettext('Are you sure you want to delete the post?'))
+                            .then(function() {
+                                scope.removePost(post);
+                            });
+                    };
+
+                    scope.askRemoveItem = function(item, post) {
+                        modal.confirm(gettext('Are you sure you want to delete the item?'))
+                            .then(function() {
+                                if (scope.post.items.length === 1) {
+                                    scope.removePost(post);
+                                } else {
+                                    //update the post
+                                    //creating the update for items
+                                    var items = angular.copy(scope.post.items);
+                                    var index = scope.post.items.indexOf(item);
+                                    items.splice(index, 1);
+                                    //update the post
+                                    postsService.savePost(scope.post.blog, scope.post, items)
+                                        .then(function(message) {
+                                            notify.pop();
+                                            notify.info(gettext('Item removed'));
+                                        }, function(erro) {
+                                            notify.pop();
+                                            notify.error(gettext('Something went wrong'));
+                                        });
+                                }
+                            }
+                        );
+                    };
+                }
+            };
+        }
+    ])
+    .directive('lbBindHtml', [function() {
```

Comment:
You already have this directive here https://github.com/superdesk/liveblog/blob/feature/auto-updates/client/app/scripts/liveblog-edit/directives.js#L226
I suspect a bad conflict resolution